apps/engmanagement/src/pages/api/skill/[...skillRecordings].ts

```diff
@@ -3,8 +3,13 @@ import SkillRecordings, {
 } from '@skillrecordings/skill-api'
 
 import prisma from '../../../db'
+import config from '../../../config'
 
 export const skillOptions: SkillRecordingsOptions = {
+  site: {
+    title: config.defaultTitle,
+    supportEmail: process.env.NEXT_PUBLIC_SUPPORT_EMAIL,
+  },
```

Comment:
for most libs you want to avoid passing in env variables, particularly with caching via turborepo. better hygiene to pass them in as config